cmd/docker-app/push.go

```diff
@@ -26,15 +24,13 @@ func pushCmd() *cobra.Command {
 				return err
 			}
 			defer app.Cleanup()
-			dgst, err := packager.Push(app, opts.namespace, opts.tag, opts.repo)
+			dgst, err := packager.Push(app, opts.tag)
 			if err == nil {
 				fmt.Println(dgst)
 			}
 			return err
 		},
 	}
-	cmd.Flags().StringVar(&opts.namespace, "namespace", "", "Namespace to use (default: namespace in metadata)")
 	cmd.Flags().StringVarP(&opts.tag, "tag", "t", "", "Tag to use (default: version in metadata)")
```

Comment:
Description should be modified to something like "Target registry reference (default is <name>:<version> from metadata)"

Reply:
Done